config/zookeeper_persister_test.go

```diff
@@ -296,8 +297,8 @@ func waitOrTimeout(c <-chan struct{}, timeout time.Duration) {
 	defer ticker.Stop()
 
 	select {
-	case <- c:
-	case <- ticker.C:
+	case <-c:
```

Comment:
Pardon my ignorance of the Golang channel operator, but is there a semantic difference here? Or just style cleanup?  Looking around at example code I see it both with and without the extra ' '.

Reply:
Style cleanup. It should be a separate PR, really...